pkg/fileutil/files.go

```diff
@@ -326,9 +334,8 @@ func UploadFile(args *UploadFileArguments) error {
 		}
 
 		// set headers
-		for k, v := range headers {
-			req.Header.Set(k, v)
-		}
+		setHeaders(req, headers)
+
 		client := &http.Client{}
 		response, err := client.Do(req)
```

Comment:
I only tested for no description. I was unable to fully test the description. When logging the request body just before and after this line `response, err := client.Do(req)`, I noticed that no data is logged after the request is sent.
Given this, simulating and testing errors for the description field and even file did not quite work.